vllm/platforms/rocm.py

```diff
@@ -20,12 +19,17 @@
 class RocmPlatform(Platform):
     _enum = PlatformEnum.ROCM
 
-    @staticmethod
+    @classmethod
+    def is_cuda_alike(cls) -> bool:
+        return True
```

Comment:
I think this can be moved to the `Platform` class.

Reply:
It's already defined in `Platform` class. Not sure what you mean by this.

Reply:
I mean, this implementation can be moved to the `Platform` class, and you don't need to implement it in `rocm.py`. Just like `is_cuda` is implemented in `Platform` class rather than `RocmPlatform` class.

Reply:
Ah, I understand now. Let me update this.